File: 프로그래머스/0/181905. 문자열 뒤집기/문자열 뒤집기.js
function reverseOfStringPart(my_string, s, e) {
    let result = my_string.slice(0, s);
    
    for (let i = 0; i <= e - s; i++) result += my_string[e - i];
    result += my_string.slice(e + 1);
    
    return result;
}

function solution(my_string, s, e) {
    let answer = '';
    
    answer = reverseOfStringPart(my_string, s, e);
    
    return answer;
}

File: 프로그래머스/0/181905. 문자열 뒤집기/문자열 뒤집기.cpp
#include <string>
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, int s, int e) {
    string answer = "";
    char temp;
    int j = 0;
    for (int i = s; i < e - j; i++)
    {
        temp = my_string[i];
        my_string[i] = my_string[e - j];
        my_string[e - j] = temp;
        j++;}
    answer = my_string;
    
    return answer;
}